glossary.yml

```diff
@@ -1919,12 +1955,34 @@
       the relationships between values, and ways to access or manipulate those data, such as
       reading, altering, or writing.
 
+- slug: data_type
+  en:
+    term: "data type"
+    def: >
+      An attribute of data which defines what operations can be performed
+      and defines the range of values that can be stored in a data object
+      of a given type. Examples of common types include: boolean or logical,
+      binary, integer,  floating point numeric, character or string.     
+  ja:
+    term: "データ型"
+    def: >
+      演算を使用出来って、データー実体に入られるの設定範囲を
+      定義するの属性のこと。
+      データ型はブーリアン型 ・論理型、バイナリ型 、整数型、数値型、文字型 とか。
+
+
 - slug: data_wrangling
+  ref:
+    - data_munging
   en:
     term: "data wrangling"
     def: >
       A colloquial name for small-scale [data engineering](#data_engineering).
-
+  ja:
+    term: "データラングリング"
```

Comment:
I put the same suggestion for "data munging" here. Googling shows that "data wrangling" is much more widely used than "data munging" in Japanese.

Reply:
Thanks resolved locally by: 2facfe320721d

As mentioned above I'm not sure how to handle synonyms for glosario. Data "munging" is pretty widely used in the R community but that might be the strong NZ influence 😄